lua/astrocommunity/editing-support/nvim-regexplainer/nvim-regexplainer.lua

```diff
@@ -0,0 +1,9 @@
+return {
+  "bennypowers/nvim-regexplainer",
+  opts = {
+    auto = true,
+  },
+  keys = {
+    { "<leader>ur", "<cmd>RegexplainerToggle<cr>", desc = "Toggle Regexplainer" },
```

Comment:
you could use the `mappings = {}` instead of keys.

Reply:
I think this is wrong @luxus . The lazy.nvim syntax is `keys`